publish.html

```diff
@@ -0,0 +1,580 @@
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishPublicWeb.jpg" class="align-left" src="images/publishPublicWeb.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">For images visible to the public, the data must be in the public group.</p>
+
+<p>You remain the owner of any images in the public group, and can still change rendering settings, add ROIs or annotate them.</p>
+
+<p>You can either move data from an existing group into the public group, or you can import data directly into the public group.</p>
+
+</li>
+
+<li><p class="first">Select the project, dataset or images to be moved to the public group in the data tree in left hand pane.</p>
+
+<p>Right-click and select <span class="bold">Move to Group ...</span> in the menu.</p>
+
+<p>Select a destination for the data in terms of Group, Project and Dataset.</p>
+
+<p>The progress of the move will show in the <span class="bold">Activities</span> window.</p>
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishMoveInsight.jpg" class="align-left" src="images/publishMoveInsight.jpg" style="width: 75%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishMoveWeb.jpg" class="align-left" src="images/publishMoveWeb.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+</li>
+
+<li><p class="first">To import data directly into the public group, in the Import Location window choose the public group from the <span class="bold">Group</span> drop-down menu.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishImport.jpg" class="align-center" src="images/publishImport.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Image IDs, shown in the right pane, are used to link to images in the PUBLIC group.</p>
```

Comment:
Here you have 'PUBLIC group', elsewhere there is a mix of 'Public Group' and 'public group', not sure which this one is meant to be or if they should all be the same other than when referring to the actual name of the group used in the example.